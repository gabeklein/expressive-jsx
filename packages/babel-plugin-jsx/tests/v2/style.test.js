transform("hoisted", `
  const App = () => {
    color: red;

    <this>Hello World</this>
  }
`)

transform("element", `
  () => {
    element: {
      color: red;
    }

    <element>Hello World</element>
  }
`)