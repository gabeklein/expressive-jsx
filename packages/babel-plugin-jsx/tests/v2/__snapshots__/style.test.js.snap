// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`element 1`] = `
  () => <div className="element_5ae">Hello World</div>;
  css(\`
  	.element_5ae {
  		color: red;
  	}
  \`);
`;

exports[`hoisted 1`] = `
  const App = () => (
    <div className="App_8dv">Hello World</div>
  );
  
  css(\`
  	.App_8dv {
  		color: red;
  	}
  \`);
`;

exports[`nested element 1`] = `
  () => (
    <div>
      <div />
      <div className="bar_l7l">
        <div className="baz_5fk">Hello World</div>
      </div>
    </div>
  );
  
  css(\`
  	.bar_l7l {
  		color: red;
  	}
  	.baz_5fk {
  		color: blue;
  	}
  \`);
`;
